src/java.base/share/classes/java/util/concurrent/ForkJoinPool.java

```diff
@@ -1904,7 +1881,7 @@ else if ((v = w) == null)
                 break;
             else
                 nc = (v.stackPred & LMASK) | (c & TC_MASK);
-            if (c == (c = ctl) && c == (c = compareAndExchangeCtl(c, nc | ac))) {
+            if (c == (c = compareAndExchangeCtl(c, nc | ac))) {
```

Comment:
So the TTAS wasn't worth it on some architectures? 🤔

Reply:
Right. so just added to code path